styles.css

```diff
@@ -5,15 +5,27 @@ body {
 .container {
   margin: 0 auto;
   width: 315px;
-  height: 4810px;
+  height: 5210px;
```

Comment:
- [ ] Kindly make sure your page height is limited where the content is limited.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212950735-6b8356fc-4544-4b75-84de-55d4f1205e4e.png)

Reply:
- [ ] Also make sure your button has the `hover`, `active`, `disable` state as it is stated in the template
![image](https://user-images.githubusercontent.com/97100091/212951228-24985255-4ece-455b-90a6-b4906e289e6b.png)